aggregator/src/test/scala/com/stratio/sparkta/aggregator/CubeMakerSpec.scala

```diff
@@ -59,56 +59,61 @@ class CubeMakerSpec extends TestSuiteBase {
     val checkpointInterval = 10000
     val checkpointTimeAvailability = 600000
     val checkpointGranularity = "minute"
-    val timeDimension = "minute"
-    val timestamp = DateOperations.dateFromGranularity(DateTime.now(), checkpointGranularity)
+    val timeDimensionName = "minute"
+    val millis = DateOperations.dateFromGranularity(DateTime.now, checkpointGranularity)
+    val sqlTimestamp = new Timestamp(millis)
     val name = "cubeName"
     val operator = new CountOperator("count", Map())
     val multiplexer = false
     val defaultDimension = new DefaultField
+    val timeField = new DateTimeField
     val dimension = Dimension("dim1", "eventKey", "identity", defaultDimension)
-    val cube = new Cube(name,
-      Seq(dimension),
+    val timeDimension = Dimension("minute", "minute", "minute", timeField)
+    val cube = Cube(name,
+      Seq(dimension, timeDimension),
       Seq(operator),
       multiplexer,
       checkpointGranularity,
       checkpointInterval,
       checkpointGranularity,
       checkpointTimeAvailability)
-    val dataCube = new CubeOperations(cube, timeDimension, checkpointGranularity)
-
-    testOperation(getEventInput, dataCube.extractDimensionsAggregations, getEventOutput(timestamp), PreserverOrder)
+    val dataCube = new CubeOperations(cube, timeDimensionName, checkpointGranularity)
+    testOperation(getEventInput(sqlTimestamp), dataCube.extractDimensionsAggregations,
```

Comment:
It is a small detail and I think It is not neccesary to change for this PR. As we arranged we should set a blank line between test setup and test by itself